corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -628,12 +628,10 @@ hqDefine('geospatial/js/models', [
 
         self.clearActivePolygon = function () {
             if (self.activeSavedPolygon) {
-                self.selectedSavedPolygonId('');
```

Comment:
why was this removed?

Reply:
I realised this actually introduced a bug where selecting a saved polygon resulted in the dropdown being cleared. This is because we call `clearActivePolygon()` before adding the new saved polygon to the map.

Reply:
okay.